docs/user-guide/src/quick-start.md

```diff
@@ -0,0 +1,916 @@
+# Quick-start for Metal3
+
+This guide has been tested on Ubuntu server 22.04. It should be seen as an
+example rather than the absolute truth about how to deploy and use Metal3. We
+will cover two environments and two scenarios. The environments are
+
+1. a baremetal lab with actual physical servers and baseboard management
+   controllers (BMCs), and
+2. a virtualized baremetal lab with virtual machines and sushy-tools acting as
```

Comment:
```suggestion
1. a virtualized baremetal lab with virtual machines and sushy-tools acting as
```
Our generator supports auto numbered lists, right?

If so, I'd like to see it utilized for all numbered lists here.